README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED - use rdf-vocab
+
+This gem is deprecated;  rdf-vocab (included in the linkeddata gem) now contains RDF::Vocab::LDP.  You can find the  helper method to strip LDP triples from an RDF::Graph object in the triannon gem in lib/oa_graph_helper.rb.
+
```

Comment:
Add github link to rdf-vocab at https://github.com/ruby-rdf/rdf-vocab, so potential users know what we really mean (vs. https://github.com/projecthydra-labs/rdf-vocab).